app/pages/admin/configuration.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
```

Comment:
A lot of these things can be super generic to make it super easy to add new models to the admin page, but this component and thunks seem like are going to be copy/pasted for every type of model, I bet there is a better way that will become apparent when you add a second model

Reply:
why are we taking every single thing out of the frontend again? lol there is going to be no customization by the next 2 hackathons

Reply:
Its not taking it out of the frontend, all that logic will still be in the frontend, I just don't want to copy and paste 250 lines for every model, it doesn't have to be truly generic, but there should at least be some like utility functions for the thunks and requests (which I have already made some, at leat one for the form request), and the component can have like a parent component to derive from or something, the admin panel is a little special in that it really shouldn't change much between hackathons and making it easy to add new mdoels is more important than adding customization to models

Reply:
agreed - ultimately we should have a custom request object to inject stuff jwt/etc, which is probably not priority for mx. I'm not really on board to making generic components that depend on the the already generic mhform though (lol i already dont like how generic that is).

Reply:
My reasons for generics was the fact that for someone to change a form object in the old codebase, it would required modifying both backend, frontend, database layer and then conduct styling for frontend. I didn't want that and the way that I see it, the current method handles all of those.